src/test/java/org/eclipse/pass/schema/SchemaFetcherTest.java

```diff
@@ -0,0 +1,168 @@
+package org.eclipse.pass.schema;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.dataconservancy.pass.client.PassClient;
+import org.dataconservancy.pass.model.Repository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class SchemaFetcherTest {
+
+    private PassClient passClientMock;
+    private Repository repository1mock;
+    private Repository repository2mock;
+
+    @BeforeEach
+    void setup() {
+        passClientMock = Mockito.mock(PassClient.class);
+        repository1mock = Mockito.mock(Repository.class);
+        repository2mock = Mockito.mock(Repository.class);
+    }
+
+    @Test
+    void getSchemaFromUriTest() throws StreamReadException, DatabindException, IOException, URISyntaxException {
```

Comment:
If you don't care about checking the exceptions, you can also just throws Exception instead of listing them all out.